src/readMeManipulator.ts

```diff
@@ -80,8 +83,54 @@ export class ReadMeManipulator {
     public addSuppressionBlock(readme: string) {
         return `${readme}\n\n${this.readMeBuilder.getSuppressionSection()}`
     }
+
+
+    /**
+     * This function takes a markdown document and a list of file paths and
+     * returns the list of tags that reference these file paths. It is meant to
+     * work like https://github.com/Azure/azure-rest-api-specs/blob/master/test/linter.js
+     */
+    public getTagsForFilesChanged(markDownEx: MarkDownEx, specsChanged: string[]): string[] {
+        const codeBlocks = getTagsToSettingsMapping(markDownEx.markDown);
+        const tagsAffected = new Set<string>();
+
+        for (const tag of Object.keys(codeBlocks)) {
```

Comment:
`for (const tag in codeBlocks)` ? Or `for (const [tag, settings] of Object.entries(codeBlock))` ?